cmsdb/processes/ewk.py

```diff
@@ -54,15 +54,23 @@
 
 # NNLO cross section, based on:
 # https://twiki.cern.ch/twiki/bin/viewauth/CMS/StandardModelCrossSectionsat13TeV?rev=27
+# and for 13.6 TeV, based on:
+# https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
 
 dy_lep_m50 = dy_lep.add_process(
     name="dy_lep_m50",
     id=51100,
-    xsecs={13: const.n_leps * Number(6077.22, {
-        "integration": 1.49,
-        "scale": 0.02j,
-        "pdf": 14.78,
-    })},
+    xsecs={
+        13: Number(6077.22, {
+            "integration": 1.49,
+            "scale": 0.02j,
+            "pdf": 14.78,
+        }),
+        13.6: const.n_leps * Number(2091.7, {
+            "scale": (0.008j, 0.013j),
+            "pdf": 1.0j,
```

Comment:
```suggestion
            "pdf": 0.01j,
```

This should be 0.01j, this is a 100% pdf error now